client/src/curriculum/prev-next.tsx

```diff
@@ -0,0 +1,31 @@
+import { CurriculumDoc } from "../../../libs/types/curriculum";
+import { Button } from "../ui/atoms/button";
+
+import "./prev-next.scss";
+
+export function PrevNext({ doc }: { doc: CurriculumDoc }) {
+  return (
```

Comment:
Or maybe destructure `{ prev, next }` with default values here to avoid repeating `doc.prevNext?` below.